v2.0/create-sequence.md

```diff
@@ -4,13 +4,18 @@ summary:
 toc: false
 ---
 
-<span class="version-tag">New in v2.0:</span> The `CREATE SEQUENCE` [statement](sql-statements.html)...
+<span class="version-tag">New in v2.0:</span> The `CREATE SEQUENCE` [statement](sql-statements.html) creates a new sequence in a database. Use a sequence to auto-increment integers in a table.
 
 <div id="toc"></div>
 
+## Considerations
+
+- Using a sequence is slower than using the `SERIAL` data type– incrementing a sequence requires a write to persistent storage, whereas generating a new `SERIAL`value does not. Use the `SERIAL` data type unless a sequence is preferred or required.
+- A column that uses a sequence can have a gap if a transaction increments the sequence and is then rolled back. Sequence updates are committed immediately and aren't rolled back along with their containing transaction. This is done to avoid blocking concurrent transactions that use the same sequence.
```

Comment:
Not sure what this means. So is there a missing value in the column, or is there an erroneous value for the sequence?

Reply:
this is referring to a gap in increment values. for example, if your sequence increments by 1, it would number the rows as 1, 2, 3, 4, 5... but if the update is rolled back, you could end up with 1, 2, 3, 5 (a gap between 3 and 5). do you think this needs to be clarified?

Reply:
Ohh! Yes, I think it needs to be clarified. The way it reads now, it seems like there would be a gap in the column, like a missing value in the column -not a gap in sequences.

Reply:
ok, edited to hopefully be clearer